packages/osd-plugin-helpers/src/cli.ts

```diff
@@ -118,5 +121,60 @@ export function runCli() {
         }
       },
     })
+    .command({
+      name: 'version',
+      description: `
+        Updates the version to a provided parameter, or syncs it with the version of OpenSearch Dashboards
+      `,
+      flags: {
+        boolean: ['sync'],
+        string: ['set'],
+        help: `
+          --${VERSION_PARAM_MATCH_DASHBOARDS}        Update the versions to match Dashboards' (default)
+          --${VERSION_PARAM_MATCH_INPUT}         Update the version to a specific value
+        `,
+      },
+      async run({ log, flags }) {
+        const pluginDir = await findOpenSearchDashboardsJson(process.cwd());
+        if (!pluginDir) {
+          throw createFailError(
+            `Unable to find OpenSearch Dashboards Platform plugin in [${process.cwd()}] or any of its parent directories. Has it been migrated properly? Does it have a opensearch_dashboards.json file?`
+          );
+        }
+
+        let versionValue = flags[VERSION_PARAM_MATCH_INPUT];
+        if (versionValue !== undefined) {
+          if (typeof versionValue !== 'string')
+            throw createFlagError(`expected a single --${VERSION_PARAM_MATCH_INPUT} flag`);
+          if (versionValue && !/^\d+\.\d+\.\d+(-\S+)?$/.test(versionValue))
+            throw createFlagError(
+              `expected a valid version following the --${VERSION_PARAM_MATCH_INPUT} flag`
```

Comment:
An example of a properly-formatted string or validation rules would be very helpful in this error.